.github/workflows/run-on-all-test-accounts.yml

```diff
@@ -0,0 +1,133 @@
+name: Run On All Test Accounts
+on:
+  workflow_dispatch:
+    inputs:
+      script_address:
+        required:  true
+        description: 'Specify the script to run on all the accounts'
```

Comment:
Could you please add an example here? I'm assuming it would be a relative path, right? something like `src/scripts/my-script.sh`?